src/main/kotlin/com/github/aivancioglo/resttest/http/Response.kt

```diff
@@ -41,54 +46,81 @@ abstract class Response() {
     }
 
     /**
-     * Making responseSpecification validation.
+     * Making response validation.
      *
-     * @param verifiers for responseSpecification validation.
+     * @param verifiers for response validation.
      */
     @SafeVarargs
     fun assertThat(verifier: Verifier, vararg verifiers: Verifier) = printFailuresIfExist(verifier, *verifiers)
 
     /**
-     * Making responseSpecification validation.
+     * Making response validation.
+     *
+     * @param verifiers for response validation.
+     */
+    @SafeVarargs
+    fun assertThat(verifiers: Array<Verifier>) = printFailuresIfExist(*verifiers)
+
+    /**
+     * Making response validation.
      *
-     * @param code of responseSpecification.
-     * @param verifiers for responseSpecification validation.
+     * @param code of response.
+     * @param verifiers for response validation.
      */
     @SafeVarargs
     fun assertThat(code: Int, vararg verifiers: Verifier) = printFailuresIfExist(statusCode(code), *verifiers)
 
     /**
-     * Making responseSpecification validation.
+     * Making response validation.
      *
-     * @param statusCode of responseSpecification.
-     * @param verifiers for responseSpecification validation.
+     * @param statusCode of response.
+     * @param verifiers for response validation.
      */
     @SafeVarargs
     fun assertThat(statusCode: StatusCode, vararg verifiers: Verifier) = printFailuresIfExist(
             statusCode(statusCode), *verifiers)
 
     /**
-     * Making responseSpecification validation.
+     * Making response validation.
      *
-     * @param code of responseSpecification.
-     * @param schema for responseSpecification validation.
-     * @param verifiers for responseSpecification validation.
+     * @param statusCodes of response.
+     * @param verifiers for response validation.
+     */
+    @SafeVarargs
+    fun assertThat(statusCodes: Pair<StatusCode, StatusCode>, vararg verifiers: Verifier) = printFailuresIfExist(
+            statusCode(statusCodes), *verifiers)
+
+    /**
+     * Making response validation.
+     *
+     * @param code of response.
+     * @param schema for response validation.
+     * @param verifiers for response validation.
```

Comment:
@param code Status code of response.
@param schema Schema for response validation.
@param verifiers Verifiers for response validation.